hash/pce.xml

```diff
@@ -4502,4 +4502,29 @@ https://mametesters.org/view.php?id=6597
 		</part>
 	</software>
 
+	<software name="acardduo" supported="no">
+		<description>Arcade Card Duo</description>
+		<year>1994</year>
+		<publisher>NEC</publisher>
+		<part name="cart" interface="pce_cart">
+			<feature name="slot" value="acard_duo" />
+			<dataarea name="rom" size="1">
+				<!-- bios not included -->
+			</dataarea>
+		</part>
+	</software>
+
```

Comment:
Let's put this as a rejected flag so you get it. 
It's an internal slot option, because NEC loves to make external and internal variants, it doesn't need to live as a fake SW entry. Either emulate it as a base `GAME` machine or get rid of it.